extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureLoadSpec.java

```diff
@@ -31,24 +31,24 @@
 public class AzureLoadSpec implements LoadSpec
 {
   @JsonProperty
-  private final String storageDir;
+  private final String blobPath;
 
   private final AzureDataSegmentPuller puller;
 
   @JsonCreator
   public AzureLoadSpec(
-      @JacksonInject AzureDataSegmentPuller puller,
-      @JsonProperty("storageDir") String storageDir
```

Comment:
I think having the container be part of the loadSpec would help future-proof azure deep-storage module and avoid having to introduce ugly backwards-compatibility fixes when we want to support multiple containers or moving data between containers.